core/api/src/graphql/public/types/object/business-account.ts

```diff
@@ -47,6 +49,15 @@ const BusinessAccount = GT.Object({
         domainAccount.defaultWalletId,
     },
 
+    defaultWallet: {
+      type: GT.NonNull(PublicWallet),
+      resolve: (source) =>
+        Wallets.getWalletForAccountById({
```

Comment:
errors here need to be thrown?

Reply:
Done https://github.com/GaloyMoney/galoy/pull/3697/commits/6eea1818b188f77c802ca35c8ec22609a2f14645